docs/learn-staking.md

```diff
@@ -182,23 +185,19 @@ Polkadot and the cryptography behind it [see here](learn-keys).
 
 ## Validators and nominators
 
-Since validator slots will be limited, most of those who wish to stake their DOT and contribute
-economic security to the network will be nominators. Validators do most of the heavy lifting: they
-produce new block candidates in BABE, vote and come to consensus in GRANDPA, validate the state
-transition function of parachains, and possibly some other responsibilities regarding data
-availability and [XCMP](learn-crosschain). Nominators, on the other hand, do not need to do anything
-once they have bonded their DOT. The experience of the nominator is similar to "set it and forget
-it," while the validator will be doing active service for the network by performing the critical
-operations. For this reason, the validator has certain privileges regarding the payout of the
-staking mechanism and will be able to declare its own allocation before the share is divided to
-nominators.
+Since validator slots are limited, most of those who wish to stake their DOT and contribute economic
+security to the network will be nominators.
 
-![staking](assets/NPoS/article-2.png)
+Validators do most of the heavy lifting: they produce new block candidates in BABE, vote and come to
+consensus in GRANDPA, validate the state transition function of parachains, and possibly some other
```

Comment:
Since this is on a 'learn' page would it be worth including SASSAFRAS somewhere?

Reply:
No, it's not live yet or being advertised, so no need to mention it yet